server-sent-events/Sources/App/Application+build.swift

```diff
@@ -14,9 +14,19 @@ public protocol AppArguments {
     var logLevel: Logger.Level? { get }
 }
 
-// Request context used by application
+/// A custom request context used by application
+/// The request context can store any data that is associated with the request
+/// This could include things like the authenticated user, a JWT token, or other information derived
+/// from the request such as their IP address, or user agent.
+/// 
+/// The Request Context can be accessed and modified by any middleware or route handler that has access to the request.
```

Comment:
The request context isn't available from the request, so having access to the request doesn't automatically mean you have access to the context.

Reply:
Right, that's I guess not guaranteed once you pass only that around. In our APIs I don't think we have anything that passes the Request but not the context right?